test-data/unit/python2eval.test

```diff
@@ -401,3 +401,13 @@ def f():  # no annotation
 path = 'test'
 path = os.path.join(f(), 'test.py')
 [out]
+
+[case testOverloadRelpathRegression]
```

Comment:
This test case seems unnecessary as we have two test cases that cover this already.